jenkins/add-docker-build-job.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -lt 7 ]; then
+    echo ""
+    echo "Usage: ./add-docker-build-job.sh <jenkins url>"
+    echo "                                 <jenkins user>"
+    echo "                                 <jenkins password>"
+    echo "                                 <git url>"
+    echo "                                 <container registry url>"
+    echo "                                 <container registry user name>"
+    echo "                                 <container registry password>"
+    echo "                                 [opt: credentials id]"
+    echo "                                 [opt: credentials description]"
+    echo "                                 [opt: job short name]"
+    echo "                                 [opt: job display name]"
+    echo "                                 [opt: job description]"
+    echo "                                 [opt: container tag prefix]"
```

Comment:
The README says it "must have" the container tag prefix, but here it's optional. We should be consistent either way